genie-web/src/main/resources/db/migration/postgresql/V4_0_0__Genie_4.sql

```diff
@@ -150,7 +150,9 @@ ALTER TABLE jobs
   ADD COLUMN agent_version                    VARCHAR(255)  DEFAULT NULL,
   ADD COLUMN agent_pid                        INT           DEFAULT NULL,
   ADD COLUMN claimed                          BOOLEAN       DEFAULT FALSE NOT NULL,
-  ADD COLUMN v4                               BOOLEAN       DEFAULT FALSE NOT NULL;
+  ADD COLUMN v4                               BOOLEAN       DEFAULT FALSE NOT NULL,
+  ADD COLUMN requested_archive_location_prefix VARCHAR(1024) DEFAULT NULL;
```

Comment:
Since your column here is so long (I still recommend removing `_prefix`) you should fix the formatting so it all lines up in the other rows.

Reply:
Shall I rename to `requested_archive_location_root` @tgianos based on the previous explanation I provided

Reply:
eh, I'm not going to pull my hair out either way. `requested_archive_location` vs. `archive_location` to me represents a concrete ask vs. realized value that have expected differences. That may just be my domain knowledge though. I don't think `root` or `prefix` adds any real information or value and is just more characters all over, but again it's not something worth killing ourselves over. I just go for simplicity where possible.